lib/estemplate.js

```diff
@@ -1,161 +1,186 @@
 const types = require('./operatorPrecedence')
 const notNull = require('./utilityFunctions').notNull
 const notUndefined = require('./utilityFunctions').notUndefined
-let estemplate = {}
+const estemplate = {}
```

Comment:
Let's not change all `let` bindings to `const`

Reply:
let it be then, let's add a `.eslintrc` in a future commit, to address this if required